lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
I'm for 2nd option.

On a side note, I think `__using__` in `calendar` is intrusive since people calling `use Calendar` when the only thing they need is a single `alias Calendar.X` most of the times. When I've seen `use Calendar` for first time I thought some magic is involved.
Same thing with the `timex`. :bowtie: